### ISSUE TITLE
Accuracy problem

### ISSUE THREAD

**Issue by RRRRendy** [peterlipan/ASD_GP_GCN#5]
Great job! I used the commands

python download_ABIDE.py
python main.py
But I got the results:

GCN 10 fold test set results, loss = 0.691561, accuracy = 0.517241
GCN 10 fold val set results, loss = 0.632226, accuracy = 0.576923
Mean Accuracy: 0.522257

I haven't changed any parameters, and I would like to ask you if there might be other possible issues that are preventing me from replicating your results.

**Comment by peterlipan** [comment]
Hi! I'm sorry that this project is no longer being maintained, and I can hardly remember the implementation details. Maybe you can check the dataset (I am not sure if you can retrieve the same dataset using download_ABIDE.py), preprocessing, and running environments.

**Comment by RRRRendy** [comment]
您的邮件我已收到，谢谢合作，祝您取得理想成绩

**Comment by RRRRendy** [comment]
Do you mean to say that your dataset was not obtained through 'python download_ABIDE.py'? May I have access to the data corresponding to your results?

**Comment by RRRRendy** [comment]
Can I access the dataset you use?

**Comment by 15265792229** [comment]
您好，这篇论文是否已经发表？
